### ISSUE TITLE
Unsupported new ANON const CVs

### ISSUE THREAD

**Issue by atoomic** [rurban/perl-compiler#246]
we probably do not want to support this, but we cannot compile a code using that experimental featureon 5.22

```perl
no warnings "experimental::const_attr";
print qq{ok\n} if &{sub () : const { 42 }} == 42;
```
````
> perl -e 'no warnings "experimental::const_attr"; print qq{ok\n} if &{sub () : const { 42 }} == 42;'
ok
> perlcc -e 'no warnings "experimental::const_attr"; print qq{ok\n} if &{sub () : const { 42 }} == 42;'
> ./a.out
Segmentation fault
````

**Comment by rurban** [comment]
No we should

**Comment by toddr** [comment]
./t/testc.sh 246

**Comment by rurban** [comment]
This has nothing to do with experimental::const_attr.
Duplicate of #305

**Comment by rurban** [comment]
anon const is just a CONSTSUB. This failed in earlier perl versions.
Fixed with `anonconst-gh246` 3c6ffa2 but this broke many other tests...

**Comment by rurban** [comment]
Fixed in master with dfb0a35bace5d8bc9c6762b8568c926dfba96b07
